assets/scss/3-components/2-product-list.scss

```diff
@@ -0,0 +1,86 @@
+#products {
+  .content-center {
+    margin-bottom: 20px;
+    @media all and (min-width: 600px) {
+      margin-bottom: 30px;
+    }
+  }
+}
+
+.products-list {
+  @include flex(row, flex-start, flex-start, wrap);
+}
+
+.products-list__item {
+  padding: 16px 8px;
+  border: 1px solid $gray;
+  border-radius: 2px;
+  width: calc(50% - 10px);
+  margin-bottom: 40px;
+  text-decoration: none;
+  transition: border 0.3s ease;
+  will-change: border;
+  margin-bottom: 20px;
+
+  @media all and (max-width: 599px) {
+    &:nth-child(2n) {
+      margin-left: 20px;
```

Comment:
Maybe `gap` property on the `flex` parent element would cover your needs for spacing between elements?